demo/Makefile.am

```diff
@@ -18,9 +18,9 @@ default-font.h: default-font.ttf
 		echo '};'; \
 	} > $@
 
-if HAVE_FREETYPE2
-if HAVE_GL
-if HAVE_GLUT
+#if HAVE_FREETYPE2
+#if HAVE_GL
+#if HAVE_GLUT
```

Comment:
This is to fix "make dist" of demo folder when these dependencies are not available. Not a great solution but to make our CI pass.